index.html

```diff
@@ -6,5 +6,22 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe 3</title>
   </head>
-  <body></body>
+  <body>
+    <nav class="navbar">
+      <img src="" alt="">
```

Comment:
can you already add the image? 
all images (including latest high quality) are in the repo now